configure

```diff
@@ -40,7 +50,7 @@ fi
 mkdir -p SITK
 (
     cd SITK &&
-    [-d SimpleITK ] ||
+    [ -d SimpleITK ] ||
```

Comment:
This should be fixed in master. Perhaps you should rebase onto master and squash the commits together?